channel_acceptor.go

```diff
@@ -196,5 +196,14 @@ func (s *ChannelAcceptor) acceptChannel(_ context.Context,
 		}, nil
 	}
 
-	return &lndclient.AcceptorResponse{Accept: true}, nil
+	if expectedChanBid.ZeroConfChannel {
```

Comment:
This should also look at the channel type bit as well (the stricter version of the protocol).

Reply:
This one is a bit tricky because I think we do not have it in the same way we had FF for channel announcement. However, LND will fail to open the channel IF we tell him "accept this channel, it's supposed to be a zeroconf one" and later on the opener does not say anything about it being a zero conf one. That's achieved by setting `ZeroConf: true` here and lndcilent sets it in the  `lnrpc.ChannelAcceptResponse`

Reply:
I'm a bit confused. What I mean here is that we want to verify the remote party actually intends to open a zero conf channel as specified in the bid order. Otherwise it seems possbile that we'll permit that any trader can open a zero conf channel here, and we'll accept it, no matter what the bid says. 

The acceptor request has a new field that can be checked: https://github.com/lightningnetwork/lnd/blob/d0d4782c3efce192b6b85160b1bd78ca7b83d528/lnrpc/lightning.proto#L930-L932.